modules/templates/conf.py_t

```diff
@@ -43,7 +43,7 @@ from recommonmark.transform import AutoStructify
 
 # Adding scripts directory to path
 sys.path.insert(0, os.path.abspath('scripts'))
-
+import blog
```

Comment:
I think this is not needed. extensions would be imported by sphinx. Please confirm this and if true remove this.